dbschema/partnership-producing-medium.esdl

```diff
@@ -0,0 +1,8 @@
+module Product {
+  type PartnershipProducingMedium extending Engagement::Child {
+    required medium: Medium;
+    required partnership: default::Partnership;
```

Comment:
Probably delete this on partnership delete too. Sorry I missed this at first.

Reply:
Er...readonly on `partnership` as well?

Reply:
Sure